docs/getting-started/using-ale/isd-generate.md

```diff
@@ -0,0 +1,147 @@
+
+# Generating an ISD with isd_generate
+
+!!! success "Goal"
+
+    Generate Instrument Support Data (ISD) for an image with ALE's `isd_generate.py`
+
+## Prerequisites
+
+You will need three things before you can generate an ISD from an image:
+
+- [Installation of ISIS](../../how-to-guides/environment-setup-and-maintenance/installing-isis-via-anaconda.md)
```

Comment:
These prefers should be more in the form of a numbered list to show you do these in order.